lstchain/tools/tests/test_tools.py

```diff
@@ -162,13 +163,24 @@ def test_create_dl3_with_config(temp_dir_observed_files, observed_dl2_file):
                 "--source-ra=83.633deg",
                 "--source-dec=22.01deg",
                 "--overwrite",
-                "--optimize-gh",
             ],
             cwd=temp_dir_observed_files,
         )
         == 0
     )
 
+@pytest.mark.private_data
+@pytest.mark.run(after="test_create_irf_full_enclosure_with_config")
```

Comment:
Remove the `mark.run` stuff, that's from pytest ordering. Use the correct fixtures instead.